tests/scripts/cute_dsl_kernels/run_custom_op_dense_gemm_swiglu.py

```
@@ -0,0 +1,430 @@
+                                   rtol=0.1)
+        print("  PASSED")
+        return True
+    except AssertionError as e:
+        print(f"  FAILED: {e}")
+        return False
+
+
+def test_fp4_output(m: int,
+                    intermediate_size: int,
+                    k: int,
+                    tolerance: float = 0.15):
```

Ruff (0.15.2)

[warning] Unused function argument: `tolerance`

(ARG001)